Cloud/AWS_TF/network.tf

```diff
@@ -25,9 +25,9 @@ resource "aws_security_group" "allow_ssh" {
 
   ingress {
     description = "SSH from VPC"
-    from_port   = 22
-    to_port     = 22
-    protocol    = "tcp"
+    from_port   = 0
```

Comment:
from_port = 0 and to_port = 0: This range (0-0) indicates all ports are allowed. Please narrow scope.